Graphs/testHelper.js

```diff
@@ -127,6 +127,39 @@ export const appendChildToElement = (element) => document.body.appendChild(eleme
 
 export const removeElement = (element) => document.body.removeChild(element);
 
+export const columnStatusTextGraphData = ($, rowNo) => {
```

Comment:
Refactor this to receive the textData object and then process further. Sending hard-coded indices doesn't make sense. The test should determine the textData object.

```
export const columnStatusTextGraphData = ($, textData) => {
const value = textData.find('span').map(
        function (i) {
            return $(this).text().trim();
        }
    ).get();
    return value.reverse();
}
```

From your test, compute current textData:
```
let textData = $('.inline-alarms').children().first(); // .next() for further nodes.
const value = columnStatusTextGraphData($, textData);
```

Reply:
Done.